optuna/storages/_rdb/storage.py

```diff
@@ -446,46 +468,50 @@ def _create_new_trial(
 
         # Retry a couple of times. Deadlocks may occur in distributed environments.
         n_retries = 0
-        with _create_scoped_session(self.scoped_session) as session:
-            while True:
+        MAX_RETRIES = 5
+        while n_retries <= MAX_RETRIES:
+            # TODO(takizawa): Ponder if this exponential backoff is useful.
+            if n_retries != 0:
+                time.sleep((2**n_retries) * (1.0 + random.random()))
```

Comment:
TBD

Reply:
I choiced this backoff timescale to mitigate congestion, but I rethinked and it is outside the scope of Optuna's responsibility. (User should scale-up the DB server) I will chance the timescale to 1/100.